modules/core/src/test/java/org/apache/ignite/internal/processors/cache/CacheMvccTxFastFinishTest.java

```diff
@@ -31,6 +33,8 @@
 public class CacheMvccTxFastFinishTest extends CacheTxFastFinishTest {
```

Comment:
Actually, fixed test is muted.
Let's fix this.

Reply:
Thanks for noting! Fixed.